src/site/content/en/blog/file-system-access/index.md

```diff
@@ -584,8 +584,9 @@ Planning to use the File System Access API on your site? Your public support hel
 features, and shows other browser vendors how critical it is to support them.
 
 - Share how you plan to use it on the [WICG Discourse thread][wicg-discourse]
-- Send a Tweet to [@ChromiumDev][cr-dev-twitter] with `#filesystemaccess` and let us know where and how
-  you're using it.
+- Send a tweet to [@ChromiumDev][cr-dev-twitter] with hashtag
```

Comment:
```suggestion
- Send a tweet to [@ChromiumDev][cr-dev-twitter] with the hashtag
```

Reply:
Made this call to action consistent in https://github.com/GoogleChrome/web.dev/pull/4722/commits/a2da5b8fc1d23310762c809a007b709e24c1a909.